src/snowflake/cli/plugins/nativeapp/utils.py

```diff
@@ -7,3 +10,58 @@ def needs_confirmation(needs_confirm: bool, auto_yes: bool) -> bool:
 
 def is_tty_interactive():
     return stdin.isatty() and stdout.isatty()
+
+
+def get_first_paragraph_from_markdown_file(file_path: Path) -> Optional[str]:
+    """
+    Reads a Markdown file at the given file path and finds the first paragraph
+
+    Parameters:
+        file_path (Path): Path to Markdown file
+
+    Returns:
+        Optional[str]: the first paragraph as a string, or None
+        if no paragraph could be found
+
+    Raises:
+        FileNotFoundError: if file_path to Markdown file does not exist
+    """
+    if not file_path.exists():
+        raise FileNotFoundError(file_path)
```

Comment:
I see the `None` fallback for line 33. But in the case this file is not found, this will throw an error instead of the `None` from the disc above. Can we make both cases `None`?